src/atoms/Tooltip/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import styles from './tooltip.module.scss';
+import classnames from 'classnames';
+
+function Tooltip ( props ) {
+  const {
+    children,
+    onClick,
+    className
+  } = props;
+
+  return (
+    <div className={ classnames(styles['tooltip'], className) }>
+      <Icon icon='tooltip' 
+        onClick={ onClick }
```

Comment:
@drewdrewthis take all the `onClick` stuff out from this component and then this LGTM